src/react-apps/applications/service-development/src/features/deploy/containers/deployToTestContainer.tsx

```diff
@@ -0,0 +1,260 @@
+import { Grid, Hidden, Paper, Typography } from '@material-ui/core';
+import { createMuiTheme, createStyles, withStyles, WithStyles } from '@material-ui/core/styles';
+import * as React from 'react';
+import { connect } from 'react-redux';
+import altinnTheme from '../../../../../shared/src/theme/altinnStudioTheme';
+import { getLanguageFromKey } from '../../../../../shared/src/utils/language';
+import postMessages from '../../../../../shared/src/utils/postMessages';
+import { makeGetRepoStatusSelector } from '../../handleMergeConflict/handleMergeConflictSelectors';
+import DeployPaper from '../components/deployPaper';
+import DeployActionDispatcher from '../deployDispatcher';
+import { makeGetCompileStatusResultSelector, makeGetCompileStatusUniqueFilenames } from '../deploySelectors';
+
+const theme = createMuiTheme(altinnTheme);
+
+// TODO: Implement multiple environment support
+const environment = 'at21';
+
+const styles = () => createStyles({
+  mainLayout: {
+    paddingTop: 20,
+    [theme.breakpoints.up('md')]: {
+      height: `calc(100vh - 110px)`,
+      overflowY: 'auto',
+      paddingLeft: theme.sharedStyles.mainPaddingLeft,
+    },
+    [theme.breakpoints.down('sm')]: {
+      height: `calc(100vh - 55px)`,
+      overflowY: 'auto',
+    },
+  },
+  ingress: {
+    paddingTop: 10,
+  },
+  deployPlaceholderStyle: {
+    marginTop: 24,
+    [theme.breakpoints.up('md')]: {
+      paddingRight: 60,
+    },
+  },
+  aboutServicePlaceholderStyle: {
+    [theme.breakpoints.up('md')]: {
+      borderLeft: '1px solid ' + theme.altinnPalette.primary.greyMedium,
+      paddingLeft: 12,
+    },
+    marginTop: 24,
+  },
+  mainGridStyle: {
+    maxWidth: 1340,
+  },
+});
+
+export enum inSyncStatus {
+  ahead = 'ahead',
+  behind = 'behind',
+  ready = 'ready',
+}
+
+export interface IDeployToTestContainerProps extends WithStyles<typeof styles> {
+  compileStatus: any;
+  compileStatusUniqueFilenames: [];
+  deploymentList: any;
+  deployStatus: any;
+  language: any;
+  masterRepoStatus: any;
+  repoStatus: any;
+}
+
+export class DeployToTestContainer extends
+  React.Component<IDeployToTestContainerProps> {
+
+  public interval: any = null;
+
+  constructor(_props: IDeployToTestContainerProps) {
+    super(_props);
+  }
+
+  public componentDidMount() {
+    const altinnWindow: any = window;
+    const { org, service } = altinnWindow;
```

Comment:
Tidier with deconstruction on the same line.
const { org, service } = window as IAltinnWIndow;

Reply:
Fixed.